test/cmocka/src/math/trig/asin_32b_fixed.c

```diff
@@ -0,0 +1,167 @@
+	-16.0428182780742645, -15.4698604643344879, -14.8969026803970337, -14.3239448666572571,
+	-13.7509870827198029, -13.1780292987823486, -12.6050714850425720, -12.0321137011051178,
+	-11.4591559171676636, -10.8861981034278870, -10.3132403194904327, -9.7402825057506561,
+	-9.1673247218132019, -8.5943669378757477, -8.0214091241359711, -7.4484513401985168,
+	-6.8754935562610626, -6.3025357425212860, -5.7295779585838318, -5.1566201448440552,
+	-4.5836623609066010, -4.0107045769691467, -3.4377467632293701, -2.8647889792919159,
+	-2.2918311953544617, -1.7188733816146851, -1.1459155976772308, -0.5729577839374542,
+	0.0000000000000000, 0.5729577839374542, 1.1459155976772308, 1.7188733816146851,
+	2.2918311953544617, 2.8647889792919159, 3.4377467632293701, 4.0107045769691467,
+	4.5836623609066010, 5.1566201448440552, 5.7295779585838318, 6.3025357425212860,
+	6.8754935562610626, 7.4484513401985168, 8.0214091241359711, 8.5943669378757477,
+	9.1673247218132019, 9.7402825057506561, 10.3132403194904327, 10.8861981034278870,
+	11.4591559171676636, 12.0321137011051178, 12.6050714850425720, 13.1780292987823486,
+	13.7509870827198029, 14.3239448666572571, 14.8969026803970337, 15.4698604643344879,
+	16.0428182780742645, 16.6157760620117188, 17.1887338459491730, 17.7616916596889496,
+	18.3346494436264038, 18.9076072275638580, 19.4805650413036346, 20.0535228252410889,
+	20.6264806389808655, 21.1994384229183197, 21.7723962068557739, 22.3453540205955505,
+	22.9183118045330048, 23.4912695884704590, 24.0642274022102356, 24.6371851861476898,
+	25.2101429998874664, 25.7831007838249207, 26.3560585677623749, 26.9290163815021515,
+	27.5019741654396057, 28.0749319493770599, 28.6478897631168365, 29.2208475470542908,
+	29.7938053607940674, 30.3667631447315216, 30.9397209286689758, 31.5126787424087524,
+	32.0856365263462067, 32.6585943102836609, 33.2315521240234375, 33.8045099079608917,
+	34.3774677217006683, 34.9504255056381226, 35.5233832895755768, 36.0963411033153534,
+	36.6692988872528076, 37.2422566711902618, 37.8152144849300385, 38.3881722688674927,
+	38.9611300826072693, 39.5340878665447235, 40.1070456504821777, 40.6800034642219543,
+	41.2529612481594086, 41.8259190320968628, 42.3988768458366394, 42.9718346297740936,
+	43.5447924435138702, 44.1177502274513245, 44.6907080113887787, 45.2636658251285553,
+	45.8366236090660095, 46.4095813930034637, 46.9825392067432404, 47.5554969906806946,
+	48.1284548044204712, 48.7014125883579254, 49.2743703722953796, 49.8473281860351562,
+	50.4202859699726105, 50.9932437539100647, 51.5662015676498413, 52.1391593515872955,
+	52.7121171653270721, 53.2850749492645264, 53.8580327332019806, 54.4309905469417572,
+	55.0039483308792114, 55.5769061148166656, 56.1498639285564423, 56.7228217124938965,
+	57.2957795262336731
+};
+
+static void test_math_trig_asin_32b_fixed(void **state)
+{
+	(void)state;
+
+	double u;
+	double v;
+	int indx;
+	int b_i;
+
+	for (indx = 0; indx < 201; ++indx) {
+		/* convert angle unit degrees to radians */
```

Comment:
The block below looks like its doing more that convert deg to rad ?

Reply:
Yes, That's correct because  `Q_CONVERT_FLOAT` is inaccurate in calculating degree to radian conversion.

Reply:
ok, so best to comment this so others can understand.

Reply:
Sure, I will add description.